package.json

```diff
@@ -7,7 +7,7 @@
   "repository": "pandell/SlickGrid",
   "author": "Michael Leibman <michael.leibman@gmail.com>",
   "dependencies": {
-    "jquery": "*",
+    "jquery": "^2.2",
```

Comment:
Okay, that makes sense. I set it to 2.x because that is what was [used](https://build.pandell.com/viewLog.html?tab=buildLog&buildTypeId=JV_Build&buildId=213521#_state=34&focus=103) previously. I'll test the grid with 2.1 and set it to that if I don't find any issues.